### PR TITLE
OpenSHMEM 1.4 RC1

### PR DESCRIPTION
**Pull request by jdinan**
OpenSHMEM 1.4 specification release candidate 1, incorporating section committee branches:

RaymondMichael:section/front -- RaymondMichael@238bc815
jdinan:section/management -- jdinan@fe8cfa93
nspark:section/rma-amo -- nspark@660e1c87
manjugv:section/collectives -- manjugv@0bc12bef
spotluri:section/sync -- spotluri@1b0860fe
BryantLam:section/back -- BryantLam@f614b8d5

PDF: [osh-1.4-rc1.pdf](https://github.com/openshmem-org/specification/files/1289421/osh-1.4-rc1.pdf)

### CODE REVIEW

File: content/backmatter.tex [hunk 1]
```diff
@@ -444,60 +434,61 @@ \subsection{SHMEM\_CACHE}
...
+    \CONST{\_SHMEM\_REDUCE\_SYNC\_SIZE}  & \CONST{\_SHMEM\_VENDOR\_STRING} \\
 \end{tabular}
 \end{center}
 do not adhere to the \Cstd standard's reserved identifiers and the \Cpp
 standard's reserved names.  These constants have been deprecated and replaced
 with corresponding constants of prefix \shmemprefix{} that adhere to \CorCpp{}
 and \Fortran naming conventions.
 
-\subsection{\FUNC{shmem\_fetch}, \FUNC{shmem\_set}, \FUNC{shmem\_cswap},
+\subsection{\VAR{SMA\_*} Environment Variables}
```
Comment (BryantLam) [inline content/backmatter.tex]: [Missing label from front matter conflict.](https://github.com/openshmem-org/specification/pull/120/files/238bc815d72d7f5bcc5ca6dc1655b1fc0922c3bb#diff-3b37f8686e2f1bb2e5cb250aa60d8d16)

Comment (jdinan) [reply content/backmatter.tex]: Whoops, missed that in the merge. I pushed a fix to the backmatter branch, so this should be resolved. I'll let RC1 stand for now and will double check for the undefined reference in the next build.

**Comment by jdinan** [comment]
I merged in a small edit (816ea3b) from the back matter to committee to fix a missing label.  There are no other new changes.

Here is the updated PDF: [osh-1.4-rc2.pdf](https://github.com/openshmem-org/specification/files/1293674/osh-1.4-rc2.pdf)

**Comment by BryantLam** [comment]
Closes #120; Closes #110; Closes #115; Closes #113; Closes #125; Closes #118.

**Comment by jdinan** [comment]
Superseded by #187

**Pull request closed without merging**